cypress/integration/plugins/observability-dashboards/6_notebooks.spec.js

```diff
@@ -12,45 +12,60 @@ import {
   SAMPLE_URL,
   SQL_QUERY_TEXT,
   PPL_QUERY_TEXT,
-  SAMPLE_PANEL,
   BASE_PATH,
 } from '../../../utils/constants';
 
 const moveToEventsHome = () => {
-  cy.visit(`${BASE_PATH}/app/observability-dashboards#/event_analytics/`);
+  cy.visit(`${BASE_PATH}/app/observability-logs#/`);
   cy.wait(delayTime * 3);
 };
 
 const moveToPanelHome = () => {
-  cy.visit(`${BASE_PATH}/app/observability-dashboards#/operational_panels/`);
+  cy.visit(`${BASE_PATH}/app/observability-dashboards#/`);
   cy.wait(delayTime * 3);
 };
+const moveToTestNotebook = () => {
+  cy.visit(`${BASE_PATH}/app/observability-notebooks#/`, {
+    timeout: delayTime * 3,
+  });
+  cy.get('.euiTableCellContent')
+    .contains(TEST_NOTEBOOK, {
+      timeout: delayTime * 3,
+    })
+    .click();
+};
 
 describe('Adding sample visualization', () => {
   it('Add sample observability data', () => {
-    moveToPanelHome();
-    cy.get('.euiButton__text').contains('Actions').trigger('mouseover').click();
+    moveToEventsHome();
+    cy.get('button[data-test-subj="eventHomeAction"]')
+      .trigger('mouseover')
+      .click();
     cy.wait(100);
-    cy.get('.euiContextMenuItem__text')
-      .contains('Add samples')
+    cy.get('button[data-test-subj="eventHomeAction__addSamples"]')
       .trigger('mouseover')
       .click();
     cy.wait(100 * 3);
     cy.get('.euiModalHeader__title[data-test-subj="confirmModalTitleText"]')
       .contains('Add samples')
       .should('exist');
     cy.wait(100);
-    cy.get('.euiButton__text').contains('Yes').trigger('mouseover').click();
-    cy.get('.euiTableCellContent', { timeout: delayTime })
-      .contains(SAMPLE_PANEL)
-      .should('exist');
+
+    cy.get('button[data-test-subj="confirmModalConfirmButton"]')
+      .trigger('mouseover')
+      .click();
+    cy.wait(100 * 5);
```

Comment:
these arbitrary waits seem unnecessary  like i would increase the timeout on the toast below and it will tell cypress to keep looking at the DOM

Reply:
Thanks, will remove these unnecessary waits in the coming PRs post 2.7.